docker-compose/7.1.N-compose.yaml

```diff
@@ -194,15 +194,15 @@ services:
     mem_limit: 2g
     environment:
       # Solr needs to know how to register itself with Alfresco
-      - SOLR_ALFRESCO_HOST=alfresco
-      - SOLR_ALFRESCO_PORT=8080
+      SOLR_ALFRESCO_HOST: alfresco
+      SOLR_ALFRESCO_PORT: 8080
```

Comment:
why? the most current notation is with the equal sign

Reply:
of course I just looked at postgres 😀 now I remember that kics was warning about that  but only for dockerfiles.

at this point if postgres is really the last one you could align that as well